core/chains/evm/config/toml/defaults/BOB_Mainnet.toml

```diff
@@ -0,0 +1,27 @@
+ChainID = '60808'
+# OP stack https://docs.gobob.xyz/learn/introduction/op-stack
+ChainType = 'optimismBedrock'
+# finality_depth was: ~850
+FinalityDepth = 900
+# block_time was:  2s
+LogPollInterval = '5s'
+
+# finality_depth * block_time / 60 secs = ~30  min (finality time)
+NoNewFinalizedHeadsThreshold = '35m'
+
+FinalityTagEnabled = true
+
+[GasEstimator]
```

Comment:
Dont we still need to set the maxValue we are ready to pay for these chains?

Reply:
@simsonraj 
According to the documentation [here](https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/gas/docs/FEE_HISTORY_ESTIMATOR.md#configs), the gas estimator requires fewer configuration settings compared to other estimators. 
It appears that the maximum value we’re willing to pay can be indirectly controlled through the `RewardPercentile` config (which has a default). This setting effectively caps how much we’re willing to bump the transaction’s price, acting as a sanity check to prevent excessive gas increases if the RPC is unresponsive.
`it's the maximum percentile we're willing to bump a transaction's price. This is used as a sanity check method in order to avoid excessive gas bumping when an RPC is not responding.`